src/software/backend/backend.h

```diff
@@ -23,10 +23,4 @@ class Backend : public Subject<SensorMsg>,
     Backend() = default;
 
     virtual ~Backend() = default;
-
-    // Delete the copy and assignment operators because this class really shouldn't need
-    // them and we don't want to risk doing anything nasty with the internal
-    // multithreading this class potentially uses
-    Backend& operator=(const Backend&) = delete;
-    Backend(const Backend&)            = delete;
```

Comment:
Are we copying Backends now?

Reply:
No, these are deleted as part of ThreadedObserver now so I opted to remove them here. Would you rather I keep them?

Reply:
It's fine either way, I just don't see the point in removing them here

Reply:
I think I agree with removing them, because they are really only here to protect against modification because this class extends `ThreadedObserver`, which now "protects" itself.